### DISCUSSION TITLE
Synced, but cannot plot

### DISCUSSION THREAD

**Discussion by randyW1** [Chia-Network/chia-blockchain#7220] [Support]
Hello,
I would like to ask for a help. We have installed the new version of chia app (1.2.0) to both our computers. On one of our computers works just fine and we are able co create new plots, but the second one doesn't let us to either enter Plot nor Farm section - see atachment (not sure how to call it, 

![Snímek obrazovky 2021-07-08 143013](https://user-images.githubusercontent.com/87130604/124921823-1bbaa680-dff9-11eb-98b2-f46115bbf3d2.png)
![Snímek obrazovky 2021-07-08 143038](https://user-images.githubusercontent.com/87130604/124921825-1cebd380-dff9-11eb-9f82-7d01bf6c6087.png)

the wheel just keeps turning and does not let me create the plots).
Is there something we could do? We tried to reinstall the app, but still the same.

Thank you,

Randy

**Comment by loppefaaret** [comment]
what OS are you on ? 
is the both the full node and the wallet synced ? 
if you locate your `debug.log` under your users folder `~/.chia/mainnet/log/` do you see any reported ERROR or WARNING ?

**Reply by randyW1** [reply]
Hello loppefaaret,
thanks for such quick response. I am running on Windows. Found the folder, where should I look for an error/warning, please?
![Snímek obrazovky 2021-07-08 181904](https://user-images.githubusercontent.com/87130604/124957173-15d4bd80-e019-11eb-864d-05b3b320c2bd.png)

**Reply by loppefaaret** [reply]
yes, the file marked in this picture, open it up, and cehck if any ERROR or WARNINGS are being reported, that makes give us a clue to what is actually happening

**Comment by randyW1** [comment]
Also, I have two computers, each running on a different wifi. Both are synced and connected in the Full node. Not sure if this is what may cause the problem, but it has been like this since the beginning and we have never experienced a problem like this.

**Comment by randyW1** [comment]
Even tried a different key, won't help.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
